File: ex2.py
def odd_number_pyramid(n):
    for i in range(1, n + 1):
        current_number = 1
        for j in range(i):
            print(current_number, end=" ")
            current_number += 2
        print()  

rows=int(input("enter number of rows: "))
odd_number_pyramid(rows)

File: ex2.js
function oddNumberPyramid(n) {
    for (let i = 1; i <= n; i++) {
        let currentNumber = 1;
        let row = "";
        for (let j = 0; j < i; j++) {
            row += currentNumber + " ";
            currentNumber += 2;
        }
        console.log(row.trim());
    }
}

rows = parseInt(prompt("Enter number of rows: "));
oddNumberPyramid(rows);